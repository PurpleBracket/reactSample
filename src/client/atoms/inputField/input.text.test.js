/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import TextInput from './input.text'

describe('Headline', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<TextInput type='text' label='Input Label' placeholder='Text Input' />)
    expect(wrapper).toMatchSnapshot()
  })
})
