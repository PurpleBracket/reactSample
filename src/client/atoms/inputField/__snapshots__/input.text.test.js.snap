// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should match snapshot 1`] = `
<input
  className="input"
  placeholder="Text Input"
  type="text"
/>
`;
